CMakeLists.txt

```diff
@@ -212,18 +212,18 @@ install(DIRECTORY DESTINATION ${ZEEK_LOG_DIR})
 configure_file(zeek-path-dev.in ${CMAKE_CURRENT_BINARY_DIR}/zeek-path-dev)
 
 file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/zeek-path-dev.sh
-     "export ZEEKPATH=`${CMAKE_CURRENT_BINARY_DIR}/zeek-path-dev`\n"
-     "export ZEEK_PLUGIN_PATH=\"${CMAKE_CURRENT_BINARY_DIR}/src\":${ZEEK_PLUGIN_PATH}\n"
-     "export PATH=\"${CMAKE_CURRENT_BINARY_DIR}\":\"${CMAKE_CURRENT_BINARY_DIR}/src\":\"${CMAKE_CURRENT_BINARY_DIR}/auxil/spicy/spicy/bin\":\"${CMAKE_CURRENT_BINARY_DIR}/src/builtin-plugins/spicy-plugin/bin\":$PATH\n"
-     "export SPICY_PATH=\"`${CMAKE_CURRENT_BINARY_DIR}/spicy-path`\"\n"
-     "export HILTI_CXX_INCLUDE_DIRS=\"`${CMAKE_CURRENT_BINARY_DIR}/hilti-cxx-include-dirs`\"\n")
+     "export ZEEKPATH=$(${cmake_binary_dir}/zeek-path-dev)\n"
```

Comment:
So this is changing the backticks to `$(...)`.

I'm not quite grasping the `CMAKE_CURRENT_BINARY_DIR` to `${cmake_binary_dir}` change, but that might be my lack of CMake.

Does that change  matter? And why couldn't that be the default `CMAKE_BINARY_DIR` rather than our custom `cmake_binary_dir`? They seem to be the same anyway?

https://github.com/zeek/zeek/blob/8e0679120c9f60a03caba50268982511ef55263b/CMakeLists.txt#L173

Reply:
This is stuff I did when I was very first working on the btest changes and nothing was working (including `ExpandBackticks`). I'll see if I can reduce/remove any of it at this point.

Reply:
Oh wait, this is in zeek-path-dev.sh. Yeah, the backtick changes can be removed.

The reason for the change to the lowercase variables is because of this block:

```
if ( MSVC )
    string(REGEX REPLACE "^([A-Za-z]):/(.*)" "/\\1/\\2" bro_plugin_install_path "${bro_plugin_install_path}")
    string(REGEX REPLACE "^([A-Za-z]):/(.*)" "/\\1/\\2" cmake_binary_dir "${cmake_binary_dir}")
    string(REGEX REPLACE "^([A-Za-z]):/(.*)" "/\\1/\\2" cmake_current_binary_dir "${cmake_current_binary_dir}")
    string(REGEX REPLACE "^([A-Za-z]):/(.*)" "/\\1/\\2" cmake_install_prefix "${cmake_install_prefix}")
    string(REGEX REPLACE "^([A-Za-z]):/(.*)" "/\\1/\\2" cmake_source_dir "${cmake_source_dir}")
    string(REGEX REPLACE "^([A-Za-z]):/(.*)" "/\\1/\\2" zeek_script_install_path "${zeek_script_install_path}")
endif ()
```

Reply:
Also note that both the SPICY_PATH and HILTI_CXX_INCLUDE_DIRS are still wrong here:

```
SPICY_PATH=.:D:/repos/zeek/auxil/spicy/spicy/spicy/lib:D:/repos/zeek/auxil/spicy/spicy/hilti/lib
HILTI_CXX_INCLUDE_DIRS=D:/repos/zeek/auxil/spicy/spicy/hilti/runtime/include:D:/repos/zeek/auxil/spicy/spicy/spicy/runtime/include:D:/repos/zeek/out/build/x64-Debug-vs2019/auxil/spicy/spicy/include:D:/repos/zeek/out/build/x64-Debug-vs2019:D:/repos/zeek/out/build/x64-Debug-vs2019/src:D:/repos/zeek/out/build/x64-Debug-vs2019/src/include:D:/repos/zeek/src:D:/repos/zeek/src/include:D:/repos/zeek/auxil/broker/include/
```

That's ok though because we've already said that Spicy won't work on Windows and it's disabled entirely for right now.